components/productDetails/sections/productDetails/card/card.html

```diff
@@ -0,0 +1,65 @@
+<div
+  class="col-lg-5 col-12 product-details--card--container--image-box d-flex align-items-center justify-content-center"
+>
+  <span class="product-details--card--container--image-box--left">
+    <img src="./images/icons/vector-left.png" />
+  </span>
+  <img
+    src="./images/products/laptop1.png"
+    class="product-details--card--container--image-box--image"
+  />
+  <span class="product-details--card--container--image-box--right">
+    <img src="./images/icons/vector-right.png" />
+  </span>
+</div>
+<div
+  class="col-lg-7 col-12 product-details--card--container--content-box d-flex flex-column"
+>
+  <h2 class="product-details--card--container--content-box--title">
+    Lenovo Legion 9 Pro Ryzen 7 Octa Core 5800H - (32 GB/1 TB SSD/Windows 10/8
```

Comment:
not same in font -family
not use customize classes in col and row and container bootstrap

Reply:
done.